src/devtools/client/debugger/src/actions/event-listeners.js

```diff
@@ -103,8 +104,22 @@ export function getEventListenerBreakpointTypes() {
     dispatch({ type: "RECEIVE_EVENT_LISTENER_TYPES", categories });
 
     if (features.eventCount) {
-      const eventTypePoints = await client.fetchEventTypePoints(INITIAL_EVENT_BREAKPOINTS);
+      const { eventTypePoints, eventTypeResults } = await client.fetchEventTypePoints(
+        INITIAL_EVENT_BREAKPOINTS
+      );
+      console.log({ eventTypePoints, eventTypeResults });
       dispatch({ type: "RECEIVE_EVENT_LISTENER_POINTS", eventTypePoints });
+
+      const unwrappedFrameworkPoints = await unwrapFrameworkPoints(eventTypeResults);
+
+      if (unwrappedFrameworkPoints) {
```

Comment:
Implementation can definitely be improved here, but essentially, this checks if there's any react events to be "unwrapped" in the points that we just got. If so, it unwraps them to get the real event from the react event, then adds it to the other `eventTypePoints` from before. That means you end end up with both the react event and its corresponding unwrapped react event in the store under `eventTypePoints[eventNameHere]`.